.merlin

```diff
@@ -4,6 +4,7 @@ PKG cmdliner alcotest lwt.unix logs.fmt dns.mirage lwt.preemptive
 PKG hvsock hvsock.lwt-unix datakit-server.fs9p win-eventlog asl
 PKG ipaddr mirage-flow ppx_sexp_conv pcap-format mirage-console.unix tcpip
 PKG charrua-core dns uwt lwt threads dns-forward tar
+PKG mirage-vnetif
```

Comment:
@djs55 it looks like it built without it. Should I run `make commit` anyway to mirror the package in the local repo?